GameData/KerbalismConfig/Support/ROCapsules.cfg

```diff
@@ -503,6 +874,24 @@
 		%max_pressure = 0.35
 	}
 }
+@PART[ROC-D2CM]:NEEDS[ROCapsules,FeatureHabitat]:AFTER[RO-Kerbalism]
+{
+    @MODULE[Habitat]
+	{	
+		volume = 4.9     //guesstimate
+		surface = 5.3193     //m^2
```

Comment:
Seems definitely wrong, how can a 4.9 m^3 capsule have a 5.3 m^2 surface? Look at the values that the other capsules have, a similar shape and similar volume should result in a similar surface area.